package.json

```diff
@@ -21,7 +21,9 @@
     "react-helmet": "^5.2.0"
   },
   "devDependencies": {
-    "prettier": "^1.17.0"
+    "panoptes-client": "^2.12.1",
```

Comment:
`panoptes-client` and `pusher-js` regular (not just dev) dependencies?

Reply:
You're right. Will make the change.